test/components/radio_test.rb

```diff
@@ -0,0 +1,208 @@
+require 'component_test_helper'
+
+class RadioTest < ComponentTestCase
+  def component_name
+    "radio"
+  end
+
+  test "does not render anything if no data is passed" do
+    assert_empty render_component({})
+  end
+
+  test "throws an error if items are passed but no name is passed" do
+    assert_raise do
+      render_component(items: [
+        {
+          value: "government-gateway",
+          text: "Use Government Gateway"
+        }
+      ])
+    end
+  end
+
+  test "renders radio-group with one item" do
```

Comment:
Pedantry alert! Should you be allowed to render a radio group with only one item? It rather takes away the illusion of choice.

Reply:
I suppose not, that test is not really that useful is it. :)